src/yang/ietf-bgp-common.yang

```diff
@@ -14,6 +14,12 @@ submodule ietf-bgp-common {
     reference
       "RFC 6991: Common YANG Data Types.";
   }
+  import ietf-bfd-types {
+    prefix bfd-types;
+    reference
+      "RFC 9127, YANG Data Model for Bidirectional Forward
```

Comment:
This will have to reference rfc9127-bis.

Reply:
I'll make this XXXX for now and open a new issue for updating it after we're through the 9127-bis dance.